src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -79,30 +76,69 @@
   [pattern]
   (-> pattern (str/lower-case) (str/split #"\.") reverse vec))
 
-(def allow-all { :auth-type :allow :type :allow-all :qualifier :exact :length nil :pattern [] })
```

Comment:
Can't get rid of this because https://github.com/puppetlabs/trapperkeeper-authorization/blob/master/src/puppetlabs/trapperkeeper/authorization/acl.clj#L129